style.css

```diff
@@ -0,0 +1,78 @@
+#hero {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 200px;
+    margin-top: 50px;
+}
+#submit{
+        max-width: 150px;
+        width: 100%;
+        height: 30px;
+        margin: 15px 0;
+        border: 0;
+}
+.container{
+    max-width: 1000px;
+    width: 100%;
+    margin: 0 auto;
+}
+#features{
+    margin-top: 30px;
+}
+/*.grid{
+    display: flex;
+}*/
```

Comment:
For a better accessebility, I recommand you to not leave comments in your files. It will make it readable for whoever wants to take a look on your project.

Reply:
Beautifully said! 👍 👍 😄